src/Serilog/Enrichers/EnvironmentUserNameEnricher.cs

```diff
@@ -21,7 +21,11 @@
 namespace Serilog.Enrichers
 {
     /// <summary>
+#if USERNAMEFROMENV
```

Comment:
Hmm. I'll have to take another look at that later tonight. IIRC the only reason I did that was because it wouldn't work otherwise. But lots of other things have changed since then so that may not be true any longer.